src/graph.js

```diff
@@ -72,17 +76,31 @@ export default class Graph {
     };
   }
 
+  _getGroupByHourFunc() {
+    const now = new Date();
+    now.setHours(now.getHours() + 1);
+    now.setMinutes(0, 0, 0);
```

Comment:
I have noticed now an issue. This code is executed once when graph is being initialized. This means that "now" won't update when the `update` function is triggered. We need to fix it as well in interval grouping implementation

Reply:
I actually thought about this when I implemented the new hour function.

What do you think about moving it to a private variable and update it when the update function is called?

Reply:
Depends what do you want to move. Is it “new now date” or sth else. I would go for putting group_by as a private prop and reassign the func on the update event - it should be cheap anyway and this way you will still have all required vars in the same local scope

Reply:
you need to generate new Date on every update anyway

Reply:
Think I found a good solution, this way I could get rid of the new `_getGroupByHourFunc()` as well and reuse `_getGroupByIntervalFunc()`